project/src/main/webapp/candidate.html

```diff
@@ -8,9 +8,20 @@
     <link rel="stylesheet" href="style.css">
     <script src="config.js"></script>
     <script src="script.js"></script>
+    <script src="common.js"></script>
```

Comment:
The name "common" might be changed to something more descriptive of the functionality, such as "navigation"